libs/langchain/langchain/callbacks/tracers/evaluation.py

```diff
@@ -81,9 +81,12 @@ def __init__(
         )
         self.client = client or _get_client()
         self.evaluators = evaluators
-        self.executor = ThreadPoolExecutor(
-            max_workers=max(max_workers or len(evaluators), 1)
-        )
+        max_workers = max_workers if max_workers is not None else len(evaluators)
+        if not max_workers:
+            self.executor = None
```

Comment:
So this means don't use a ThreadPoolExecutor only if 
- user passes in max_workers = 0
- user passes in 0 evaluators
Why?